description: fix gcc warnings about enum comparisons
author: Michael Gilbert <mgilbert@debian.org>

--- a/chrome/browser/ui/views/passwords/password_generation_popup_view_views.cc
+++ b/chrome/browser/ui/views/passwords/password_generation_popup_view_views.cc
@@ -50,11 +50,11 @@ class PasswordGenerationPopupViewViews::
     BuildColumnSet(layout);
     layout->StartRow(views::GridLayout::kFixedSize, 0);
 
+    int text_style = STYLE_SECONDARY;
+    if (state == PasswordGenerationPopupController::kOfferGeneration)
+      text_style = views::style::STYLE_PRIMARY;
     layout->AddView(autofill::CreateLabelWithColorReadabilityDisabled(
-        suggestion, ChromeTextContext::CONTEXT_BODY_TEXT_LARGE,
-        state == PasswordGenerationPopupController::kOfferGeneration
-            ? views::style::STYLE_PRIMARY
-            : STYLE_SECONDARY));
+        suggestion, ChromeTextContext::CONTEXT_BODY_TEXT_LARGE, text_style));
 
     DCHECK(!password_label_);
     password_label_ = autofill::CreateLabelWithColorReadabilityDisabled(
--- a/components/viz/common/gl_helper.cc
+++ b/components/viz/common/gl_helper.cc
@@ -477,7 +477,7 @@ void GLHelper::CopyTextureToImpl::Cancel
 }
 
 bool GLHelper::CopyTextureToImpl::IsBGRAReadbackSupported() {
-  if (bgra_support_ == BGRA_PREFERENCE_UNKNOWN) {
+  if (bgra_support_ == BGRA_SUPPORT_UNKNOWN) {
     bgra_support_ = BGRA_NOT_SUPPORTED;
     if (auto* extensions = gl_->GetString(GL_EXTENSIONS)) {
       const std::string extensions_string =
--- a/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
+++ b/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
@@ -2064,11 +2064,13 @@ int BookmarkBarView::GetIndexForButton(v
 
 SkColor BookmarkBarView::GetBookmarkBarTextColor() {
   const ui::ThemeProvider* theme_provider = GetThemeProvider();
-  int background_color_id =
+
+  SkColor background_color =
       bookmark_bar_state_ == BookmarkBar::DETACHED
-          ? ThemeProperties::COLOR_DETACHED_BOOKMARK_BAR_BACKGROUND
-          : ThemeProperties::COLOR_TOOLBAR;
+          ? theme_provider->GetColor(ThemeProperties::COLOR_DETACHED_BOOKMARK_BAR_BACKGROUND)
+          : theme_provider->GetColor(ThemeProperties::COLOR_TOOLBAR);
+
   return color_utils::GetColorWithMinimumContrast(
       theme_provider->GetColor(ThemeProperties::COLOR_BOOKMARK_TEXT),
-      theme_provider->GetColor(background_color_id));
+      background_color);
 }
