--- a/third_party/skia/src/ports/SkFontHost_FreeType.cpp
+++ b/third_party/skia/src/ports/SkFontHost_FreeType.cpp
@@ -132,9 +132,9 @@ public:
         : fGetVarDesignCoordinates(nullptr)
         , fGetVarAxisFlags(nullptr)
         , fLibrary(nullptr)
-        , fIsLCDSupported(false)
+        , fIsLCDSupported(true)
         , fLightHintingIsYOnly(false)
-        , fLCDExtra(0)
+        , fLCDExtra(2)
     {
         if (FT_New_Library(&gFTMemory, &fLibrary)) {
             return;
