--- a/chrome/test/chromedriver/BUILD.gn
+++ b/chrome/test/chromedriver/BUILD.gn
@@ -312,8 +312,7 @@ source_set("lib") {
     "//base/third_party/dynamic_annotations",
     "//build:branding_buildflags",
     "//build:chromeos_buildflags",
-    "//chrome/common:constants",
-    "//chrome/common:version_header",
+    "//chrome/common",
     "//chrome/test/chromedriver/constants:version_header",
     "//components/crx_file",
     "//components/embedder_support",
--- a/tools/v8_context_snapshot/BUILD.gn
+++ b/tools/v8_context_snapshot/BUILD.gn
@@ -82,6 +82,7 @@ if (use_v8_context_snapshot) {
       sources = [ "v8_context_snapshot_generator.cc" ]
 
       deps = [
+        "//chrome/common",
         "//gin:gin",
         "//mojo/core/embedder",
         "//services/service_manager/public/cpp",
