--- a/build/toolchain/win/toolchain.gni
+++ b/build/toolchain/win/toolchain.gni
@@ -58,6 +58,9 @@ template("single_msvc_toolchain") {
       toolchain_is_clang = is_clang
     }
 
+    # Suppress unread variable error
+    toolchain_is_clang = toolchain_is_clang
+
     # When the invoker has explicitly overridden use_goma or cc_wrapper in the
     # toolchain args, use those values, otherwise default to the global one.
     # This works because the only reasonable override that toolchains might
@@ -192,15 +195,11 @@ template("single_msvc_toolchain") {
 
     # Disabled with cc_wrapper because of
     # https://github.com/mozilla/sccache/issues/1013
-    if (toolchain_is_clang && toolchain_cc_wrapper == "") {
       # This flag omits system includes from /showIncludes output, to reduce
       # the amount of data to parse and store in .ninja_deps. We do this on
       # non-Windows too, and already make sure rebuilds after winsdk/libc++/
       # clang header updates happen via changing command line flags.
-      show_includes = "/showIncludes:user"
-    } else {
       show_includes = "/showIncludes"
-    }
 
     tool("cc") {
       precompiled_header_type = "msvc"
