--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1514,7 +1514,7 @@ config("compiler_deterministic") {
 }
 
 config("clang_revision") {
-  if (is_clang && clang_base_path == default_clang_base_path) {
+  if (false) {
     update_args = [
       "--print-revision",
       "--verify-version=$clang_version",
--- a/build/toolchain/toolchain.gni
+++ b/build/toolchain/toolchain.gni
@@ -39,7 +39,7 @@ if (generate_linker_map) {
 
 declare_args() {
   if (llvm_force_head_revision) {
-    clang_version = "18"
+    clang_version = "17"
   } else {
     # TODO(crbug.com/1467585): Remove in the next clang roll
     clang_version = "17"
