--- a/build/toolchain/win/toolchain.gni
+++ b/build/toolchain/win/toolchain.gni
@@ -204,15 +204,11 @@ template("msvc_toolchain") {
 
     # Disabled with cc_wrapper because of
     # https://github.com/mozilla/sccache/issues/1013
-    if (toolchain_is_clang && toolchain_cc_wrapper == "") {
       # This flag omits system includes from /showIncludes output, to reduce
       # the amount of data to parse and store in .ninja_deps. We do this on
       # non-Windows too, and already make sure rebuilds after winsdk/libc++/
       # clang header updates happen via changing command line flags.
-      show_includes = "/showIncludes:user"
-    } else {
       show_includes = "/showIncludes"
-    }
 
     tool("cc") {
       precompiled_header_type = "msvc"
