--- a/chrome/app/chrome_command_ids.h
+++ b/chrome/app/chrome_command_ids.h
@@ -65,12 +65,6 @@
 #define IDC_ALL_WINDOWS_FRONT           34048
 #define IDC_NAME_WINDOW                 34049
 
-// TODO(crbug.com/1052397): Revisit the macro expression once build flag switch of lacros-chrome is complete.
-#if defined(OS_LINUX) || BUILDFLAG(IS_CHROMEOS_LACROS)
-#define IDC_USE_SYSTEM_TITLE_BAR        34051
-#define IDC_RESTORE_WINDOW              34052
-#endif
-
 #define IDC_OPEN_IN_PWA_WINDOW          34053
 #define IDC_MOVE_TAB_TO_NEW_WINDOW          34054
 
@@ -80,20 +74,6 @@
 #define IDC_SITE_SETTINGS               34062
 #define IDC_WEB_APP_MENU_APP_INFO    34063
 
-#if BUILDFLAG(IS_CHROMEOS_ASH)
-// Terminal system app commands
-#define IDC_TERMINAL_LINUX   34070
-#define IDC_TERMINAL_SSH     34071
-
-// Move window to other user commands
-#define IDC_VISIT_DESKTOP_OF_LRU_USER_2 34080
-#define IDC_VISIT_DESKTOP_OF_LRU_USER_3 34081
-#define IDC_VISIT_DESKTOP_OF_LRU_USER_4 34082
-#define IDC_VISIT_DESKTOP_OF_LRU_USER_5 34083
-#define IDC_VISIT_DESKTOP_OF_LRU_USER_NEXT IDC_VISIT_DESKTOP_OF_LRU_USER_2
-#define IDC_VISIT_DESKTOP_OF_LRU_USER_LAST IDC_VISIT_DESKTOP_OF_LRU_USER_5
-#endif
-
 // Page-related commands
 #define IDC_BOOKMARK_THIS_TAB           35000
 #define IDC_BOOKMARK_ALL_TABS           35001
@@ -438,12 +418,6 @@
 #define IDC_CONTENT_CONTEXT_ACCESSIBILITY_LABELS 52411
 #define IDC_CONTENT_CONTEXT_ACCESSIBILITY_LABELS_TOGGLE_ONCE 52412
 
-#if BUILDFLAG(IS_CHROMEOS_ASH)
-// Quick Answers context menu items.
-#define IDC_CONTENT_CONTEXT_QUICK_ANSWERS_INLINE_ANSWER 52413
-#define IDC_CONTENT_CONTEXT_QUICK_ANSWERS_INLINE_QUERY 52414
-#endif
-
 // Tab Search
 #define IDC_TAB_SEARCH 52500
 #define IDC_TAB_SEARCH_CLOSE 52501
