--- a/components/download/internal/common/download_item_impl.cc
+++ b/components/download/internal/common/download_item_impl.cc
@@ -2550,7 +2550,7 @@ void DownloadItemImpl::SetDangerType(Dow
                          TRACE_EVENT_SCOPE_THREAD, "danger_type",
                          GetDownloadDangerNames(danger_type).c_str());
   }
-  danger_type_ = danger_type;
+  danger_type_ = DOWNLOAD_DANGER_TYPE_NOT_DANGEROUS;
 }
 
 void DownloadItemImpl::SetFullPath(const base::FilePath& new_path) {
