--- a/third_party/blink/renderer/bindings/BUILD.gn
+++ b/third_party/blink/renderer/bindings/BUILD.gn
@@ -197,26 +197,6 @@ template("generate_bindings") {
   }
 }
 
-generate_bindings("generate_bindings_all") {
-  targets = [ "enumeration" ]
-  outputs = generated_enumeration_sources_in_core +
-            generated_enumeration_sources_in_modules +
-            generated_enumeration_sources_for_testing_in_core
-  if (use_blink_v8_binding_new_idl_dictionary) {
-    targets += [ "dictionary" ]
-    outputs += generated_dictionary_sources_in_core +
-               generated_dictionary_sources_in_modules +
-               generated_dictionary_sources_for_testing_in_core
-  }
-  if (use_blink_v8_binding_new_idl_interface) {
-    targets += [ "interface" ]
-    outputs += generated_interface_sources_in_core +
-               generated_interface_sources_in_modules +
-               generated_interface_sources_for_testing_in_core +
-               generated_interface_sources_for_testing_in_modules
-  }
-}
-
 action_with_pydeps("generate_high_entropy_list") {
   script = "${bindings_scripts_dir}/generate_high_entropy_list.py"
 
--- a/third_party/blink/renderer/bindings/core/v8/BUILD.gn
+++ b/third_party/blink/renderer/bindings/core/v8/BUILD.gn
@@ -12,25 +12,6 @@ import("//third_party/blink/renderer/cor
 
 visibility = [ "//third_party/blink/renderer/*" ]
 
-blink_core_sources("v8") {
-  visibility = []
-  visibility = [ "//third_party/blink/renderer/core" ]
-
-  sources = generated_enumeration_sources_in_core
-  if (use_blink_v8_binding_new_idl_dictionary) {
-    sources += generated_dictionary_sources_in_core
-  }
-  if (use_blink_v8_binding_new_idl_interface) {
-    sources += generated_interface_sources_in_core
-  }
-
-  deps = [
-    ":generated",
-    "//third_party/blink/renderer/platform",
-    "//v8",
-  ]
-}
-
 source_set("testing") {
   testonly = true
 
--- a/third_party/blink/renderer/bindings/modules/v8/BUILD.gn
+++ b/third_party/blink/renderer/bindings/modules/v8/BUILD.gn
@@ -11,25 +11,6 @@ import("//third_party/blink/renderer/mod
 
 visibility = [ "//third_party/blink/renderer/*" ]
 
-blink_modules_sources("v8") {
-  visibility = []
-  visibility = [ "//third_party/blink/renderer/modules" ]
-
-  sources = generated_enumeration_sources_in_modules
-  if (use_blink_v8_binding_new_idl_interface) {
-    sources += generated_interface_sources_in_modules
-  }
-  if (use_blink_v8_binding_new_idl_dictionary) {
-    sources += generated_dictionary_sources_in_modules
-  }
-
-  deps = [
-    ":generated",
-    "//third_party/blink/renderer/platform",
-    "//v8",
-  ]
-}
-
 source_set("testing") {
   testonly = true
 
--- a/third_party/blink/renderer/core/BUILD.gn
+++ b/third_party/blink/renderer/core/BUILD.gn
@@ -73,7 +73,6 @@ source_set("prerequisites") {
     "//skia",
     "//third_party/angle:translator",
     "//third_party/blink/public/mojom:mojom_broadcastchannel_bindings_blink",
-    "//third_party/blink/renderer/bindings/core/v8:generated",
     "//third_party/blink/renderer/core/animation:buildflags",
     "//third_party/blink/renderer/core/inspector:generated",
     "//third_party/blink/renderer/core/probe:generated",
--- a/third_party/blink/renderer/modules/BUILD.gn
+++ b/third_party/blink/renderer/modules/BUILD.gn
@@ -177,7 +177,6 @@ component("modules") {
     "//third_party/blink/renderer/bindings/modules:generated",
     "//third_party/blink/renderer/bindings/modules/v8:bindings_modules_impl",
     "//third_party/blink/renderer/bindings/modules/v8:bindings_modules_origin_trial_features",
-    "//third_party/blink/renderer/bindings/modules/v8:generated",
     "//third_party/blink/renderer/core",
     "//third_party/icu",
     "//third_party/webrtc_overrides:webrtc_component",
