--- a/build/config/clang/BUILD.gn
+++ b/build/config/clang/BUILD.gn
@@ -218,8 +218,6 @@ clang_lib("compiler_builtins") {
     } else {
       libname = "ios"
     }
-  } else {
-    libname = "builtins"
   }
 }
 
--- a/build/config/rust.gni
+++ b/build/config/rust.gni
@@ -54,13 +54,13 @@ declare_args() {
   # To use a custom toolchain instead, specify an absolute path to the root of
   # a Rust sysroot, which will have a 'bin' directory and others. Commonly
   # <home dir>/.rustup/toolchains/nightly-<something>-<something>
-  rust_sysroot_absolute = ""
+  rust_sysroot_absolute = "../../third_party/rust-toolchain"
 
   # If you're using a Rust toolchain as specified by rust_sysroot_absolute,
   # set this to the output of `rustc -V`. Changing this string will cause all
   # Rust targets to be rebuilt, which allows you to update your toolchain and
   # not break incremental builds.
-  rustc_version = ""
+  rustc_version = "rustc 1.78.0-nightly (bb594538f 2024-02-20)"
 
   # If you're using a Rust toolchain as specified by rust_sysroot_absolute,
   # you can specify whether it supports nacl here.
