--- a/third_party/blink/renderer/bindings/scripts/generate_bindings.py
+++ b/third_party/blink/renderer/bindings/scripts/generate_bindings.py
@@ -11,6 +11,10 @@ import sys
 import web_idl
 import bind_gen
 
+# shutil.copy
+import shutil
+import os
+
 
 def parse_output_reldirs(reldirs):
     required = ['core', 'modules']
@@ -74,6 +78,11 @@ def parse_options(valid_tasks):
 
 
 def main():
+    # This fix build issue on Windows caused by missing google dependencies.
+    if os.name == 'nt':
+        if not os.path.exists('../../buildtools/win/clang-format.exe'):
+            shutil.copy('../../third_party/llvm-build/Release+Asserts/bin/clang-format.exe', '../../buildtools/win')
+
     dispatch_table = {
         'callback_function': bind_gen.generate_callback_functions,
         'callback_interface': bind_gen.generate_callback_interfaces,
