--- a/build/toolchain/win/BUILD.gn
+++ b/build/toolchain/win/BUILD.gn
@@ -187,15 +187,11 @@ template("msvc_toolchain") {
     }
 
     # Disabled with cc_wrapper because of https://github.com/mozilla/sccache/issues/1013
-    if (toolchain_args.is_clang && cc_wrapper == "") {
       # This flag omits system includes from /showIncludes output, to reduce the
       # amount of data to parse and store in .ninja_deps. We do this on non-Windows too,
       # and already make sure rebuilds after win sdk / libc++ / clang header updates happen via
       # changing commandline flags.
-      show_includes = "/showIncludes:user"
-    } else {
       show_includes = "/showIncludes"
-    }
 
     tool("cc") {
       precompiled_header_type = "msvc"
