# Disable reordering imports

--- a/chrome/BUILD.gn
+++ b/chrome/BUILD.gn
@@ -59,27 +59,6 @@ if (enable_resource_whitelist_generation
 }
 
 if (is_win) {
-  action("reorder_imports") {
-    script = "//build/win/reorder-imports.py"
-
-    # See comment in chrome_dll.gypi in the hardlink_to_output
-    # target for why this cannot be 'initial' like the DLL.
-    inputs = [ "$root_out_dir/initialexe/chrome.exe" ]
-    outputs = [ "$root_out_dir/chrome.exe" ]
-    if (symbol_level != 0) {
-      outputs += [ "$root_out_dir/chrome.exe.pdb" ]
-    }
-    args = [
-      "-i",
-      rebase_path("$root_out_dir/initialexe", root_build_dir),
-      "-o",
-      rebase_path("$root_out_dir", root_build_dir),
-      "-a",
-      current_cpu,
-    ]
-    deps = [ ":chrome_initial" ]
-  }
-
   if (!is_multi_dll_chrome) {
     # Remove the chrome_child.dll build artifacts from the build directory if
     # they're not needed to avoid packaging them in the installer.
@@ -130,21 +109,13 @@ if (!is_android && !is_mac) {
   group("chrome") {
     public_deps = [ ":chrome_initial" ]
     data_deps = [ ":chrome_initial" ]
-    if (is_win) {
-      public_deps += [ ":reorder_imports" ]
-      data_deps += [ ":reorder_imports" ]
-    }
     if (is_chromeos) {
       data_deps += [ "//sandbox/linux:chrome_sandbox" ]
     }
   }
 
   executable("chrome_initial") {
-    if (is_win) {
-      output_name = "initialexe/chrome"
-    } else {
-      output_name = "chrome"
-    }
+    output_name = "chrome"
 
     if (!is_win || is_clang) {
       # Normally, we need to pass specific flags to the linker to
--- a/chrome/test/chromedriver/BUILD.gn
+++ b/chrome/test/chromedriver/BUILD.gn
@@ -352,11 +352,6 @@ python_library("chromedriver_py_tests")
   if (is_component_build && is_mac) {
     data_deps += [ "//chrome:chrome_framework" ]
   }
-
-  if (is_win) {
-    # On Windows, the following target produces the final chrome.exe
-    data_deps += [ "//chrome:reorder_imports" ]
-  }
 }
 
 python_library("chromedriver_replay_unittests") {
--- a/tools/perf/chrome_telemetry_build/BUILD.gn
+++ b/tools/perf/chrome_telemetry_build/BUILD.gn
@@ -42,10 +42,6 @@ group("telemetry_chrome_test") {
     data_deps += [ "//chrome" ]
   }
 
-  if (is_win) {
-    data_deps += [ "//chrome:reorder_imports" ]
-  }
-
   if (is_linux) {
     data_deps += [
       "//third_party/breakpad:dump_syms($host_toolchain)",
