--- a/ungoogled-chromium/patches/series
+++ b/ungoogled-chromium/patches/series
@@ -98,4 +98,3 @@ extra/ungoogled-chromium/add-flag-for-disabling-link-drag.patch
 extra/ungoogled-chromium/add-flag-to-hide-extensions-menu.patch
 extra/ungoogled-chromium/add-flag-to-hide-fullscreen-exit-ui.patch
 extra/ungoogled-chromium/add-flag-for-incognito-themes.patch
-extra/ungoogled-chromium/add-flags-for-referrer-customization.patch
